keyboards/handwired/bad_numpad/keyboard.json

```
@@ -0,0 +1,52 @@
+{
+    "manufacturer": "LucaFrankfurt",
+    "keyboard_name": "bad_numpad",
+    "maintainer": "LucaFrankfurt",
+    "development_board": "promicro",
+    "bootloader": "atmel-dfu",
```

Either this is a `pro micro` and `bootloader should be removed (which matches readme content)

```suggestion
    "development_board": "promicro",
```

or it should be updated to use the elite_c development board defaults